gradoop-flink/src/test/java/org/gradoop/flink/model/impl/functions/epgm/RenameLabelTest.java

```diff
@@ -0,0 +1,90 @@
+/**
+ * Copyright © 2014 - 2018 Leipzig University (Database Research Group)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.gradoop.flink.model.impl.functions.epgm;
+
+import static org.junit.Assert.assertEquals;
+
+import org.gradoop.common.model.api.entities.EPGMEdge;
+import org.gradoop.common.model.api.entities.EPGMGraphHead;
+import org.gradoop.common.model.api.entities.EPGMVertex;
+import org.gradoop.common.model.impl.id.GradoopId;
+import org.gradoop.common.model.impl.pojo.EdgeFactory;
+import org.gradoop.common.model.impl.pojo.GraphHeadFactory;
+import org.gradoop.common.model.impl.pojo.VertexFactory;
+import org.gradoop.common.model.impl.properties.Properties;
+import org.gradoop.flink.model.GradoopFlinkTestBase;
+import org.gradoop.flink.model.api.functions.TransformationFunction;
+import org.junit.Test;
+
+public class RenameLabelTest extends GradoopFlinkTestBase {
+
+  @Test
+  public void testGraphHead() {
+
+    GradoopId graphId = GradoopId.get();
+    String label = "A";
+
+    EPGMGraphHead graphHead = new GraphHeadFactory().initGraphHead(graphId, label);
+
+    String newLabel = "B";
+
+    TransformationFunction<EPGMGraphHead> renameFunction = new RenameLabel<>(label, newLabel);
+
+    renameFunction.apply(graphHead, graphHead);
+
+    assertEquals(newLabel, graphHead.getLabel());
+  }
+
+  @Test
+  public void testEdge() {
+
+    GradoopId edgeId = GradoopId.get();
+    GradoopId sourceId = GradoopId.get();
+    GradoopId targetId = GradoopId.get();
+
+    String label = "A";
+    Properties props = Properties.create();
+    props.set("k1", "v1");
+    props.set("k2", "v2");
+
+    EPGMEdge edge = new EdgeFactory().initEdge(edgeId, label, sourceId, targetId);
+
+    String newLabel = "B";
+
+    TransformationFunction<EPGMEdge> renameFunction = new RenameLabel<>(label, newLabel);
+
+    renameFunction.apply(edge, edge);
+
+    assertEquals(newLabel, edge.getLabel());
```

Comment:
I guess it would make sense to test if the properties are still equal after renaming.